src/v3/src/components/OpenOktaVerifyButton/OpenOktaVerifyButton.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { FunctionComponent, h } from 'preact';
+import React from 'preact/compat';
+import { useState } from 'preact/hooks';
+
+import {
+  ButtonElement,
+  ButtonType,
+  OpenOktaVerifyButtonElement,
+  UISchemaElementComponent,
+} from '../../types';
+import { getTranslation } from '../../util';
+import Button from '../Button';
+
+type IFrameProps = {
+  src: string;
+};
+const IFrame: FunctionComponent<IFrameProps> = ({ src }) => (
+  // eslint-disable-next-line jsx-a11y/iframe-has-title
+  <iframe
+    src={src}
+    style={{
+      display: 'none',
+    }}
+  />
+);
+
+const OpenOktaVerifyButton: UISchemaElementComponent<{
+  uischema: OpenOktaVerifyButtonElement
+}> = ({ uischema }) => {
```

Comment:
Just a question, does having `FP` or `FastPass` in the component/Element name help or would that be too wordy/redundant? It may be redundant since this is still technically OV, just a slight variation.

Reply:
I think adding FP might be helpful context in the name without lengthening it too much 👍